packages/slice-machine/lib/builders/SliceBuilder/index.tsx

```diff
@@ -142,15 +123,7 @@ const SliceBuilder: React.FC<SliceBuilderProps> = ({
       />
       <FlexEditor
         sx={{ py: 4 }}
-        SideBar={
-          <SideBar
-            component={component}
-            variation={variation}
-            onScreenshot={onTakingSliceScreenshot}
-            onHandleFile={onTakingSliceCustomScreenshot}
-            imageLoading={data.imageLoading}
-          />
-        }
+        SideBar={<SideBar component={component} variation={variation} />}
```

Comment:
Is this a child element?

Reply:
sort of, it passes a component as property